Makefile

```diff
@@ -266,3 +266,126 @@ release-note:
 .PHONY: generate-release-tags
 generate-release-tags:
 	hack/generate-release-tags.sh
+
+########### Operator SDK Makefile contents
```

Comment:
we need to document how the new targets interact with established flows, and perhaps invoke them from our targets. Maybe done in a later commit? (didn't see yet).

Reply:
We invoke them from our targets.

Reply:
ok, this is done in the followup patch; perhaps it could have been more obvious with a line in the commit message.

Reply:
that a follow up patch will tweak it? I can do that, sure.